src/components/Tabs/Tabs.styles.ts

```diff
@@ -72,23 +72,28 @@ export const BulbTab = styled.div<IStyledBulb>`
     padding: 8px 16px;
     display: flex;
     align-items: center;
-    background-color: ${color.blueLight};
+    background-color: ${color.culturedBlue};
     border: 2px solid transparent;
     cursor: pointer;
     color: ${color.blue};
     ${({ isActive }) =>
         isActive &&
         css`
-            border-color: ${color.blue};
-            background-color: ${color.white};
+            border-color: ${color.blueSky};
+            background: transparent;
         `};
-    ${(props) =>
-        (props.isActive || props.isDisabled) && 'pointer-events: none;'};
+    ${(props) => props.isDisabled && 'pointer-events: none;'};
     ${(props) => props.hasMargin && 'margin-right:6px;'}
-    ${(props) => props.isDisabled && `color:${color.greyDisabled};`}
+    ${(props) =>
+        props.isDisabled && 'opacity: 0.5; background-color: transparent;'}
```

Comment:
i think transparent will fix one problem for us, but cause many more for others (maybe even in our admin as tabs are used on white and also on BlueSky i believe

Reply:
yup, you're right! Let me think about this quick and come up with a solution :)